### DISCUSSION TITLE
Paper suggestion

### DISCUSSION THREAD

**Discussion by EternalDreamer01** [Trustworthy-AI-Group/TransferAttack#93] [Ideas]
Hello,

It seems you haven't tried [CDG (2018)](https://arxiv.org/pdf/1807.06752), which is told to be transferable ;
> In this paper, we show dominant adversarial examples are effective when targeting A3C path finding, and design a Common Dominant Adversarial Examples Generation Method (CDG) to generate dominant adversarial examples against any given map

Although this paper targets RL for pathfinding, I'd be interested to see its result in your benchmark.

Best regards,

Mike

**Comment by xiaosen-wang** [comment]
It is a good paper that we miss, We welcome you to contribute it to our repo. Thanks!
